docs/develop.md

```diff
@@ -37,10 +97,7 @@ This will install all the necessities for deploying and running the service,
 including a service account and cluster role binding.
 
 ```bash
-oc new-project onboarding
-
-cd k8s/overlays/crc
-oc apply -k .
+oc apply -k k8s/overlays/crc
```

Comment:
why did you remove the command that creates a project?

Reply:
I guess you are creating it in #55

Reply:
@larsks Are you running CRC locally, as I was under the impression that you were running the code outside of a container for development.

Reply:
> Are you running CRC locally, 

Yes. 

> as I was under the impression that you were running the code outside of a container for development.

Also yes. 

That is what the updated instructions describe (how to run the code locally, outside of a container). 

You still need an OpenShift instance in which you can manipulate projects and users and so forth.